proposals/daovotion-nebairevelations-oss-web3.md

```diff
@@ -0,0 +1,77 @@
+
+For testing the truth about the health of the FOSS projects, **DAOVOTION** establishes a system for evaluating the efforts in maintaining a good quality track. With [Arbitration Trials](https://nebairevelations.com/daovotion/#arbitration-trials-and-the-quest-for-truth-6bb9bdb3-3dc9-4c63-a137-784de8b34667), Devotees from other projects could act as **Jurors** and deliberate against a particular **Egregore**. Its **Prestige Score** is affected when **Jurors** evaluate how good or bad are its developers maintaining the quality of the project, and how they're responding to the demands of its users.
+
+From these trials, developers feel the pressure for providing good quality content and pursuing excellence in their efforts, and **Devotees** become aware of the value appreciation from their investments in the project. **Arbitration Trials** motivates **Authors** (developers) and **Devotees** (investors) in working consistently on improving the quality of their productions. Also, they’d be willing to invest more money and effort in correcting issues that affect the user’s experience.
+
+Because the historical performance of the **Egregore's Prestige Score** is tracked in the blockchain, any interested party could analyze how Authors (developers) are committed to the fulfillment of the community expectations about their **Intellectual Goods** development, but also shows the degree of shared responsibility that Devotees perform when honoring their contractual obligations from DeFi applications.
+
+In that way, [DAOVOTION Tokenomics](https://nebairevelations.com/daovotion/) stimulates a healthy competence between Egregores, nurturing a dynamic market where **Authors** and **Devotees** join forces in maintaining a good reputation for their Egregores and pursuing excellence in their quality efforts, just for offering the best possible value to its users. Egregore members are encouraged to think ahead and make the right decisions for their projects, as they're aware that those Egregores with higher Prestige Score are more attractive to potential investors that are just looking for good quality collateral.
+
+### Due diligence
+
+<!-- Please answer the following due diligence questions; it's okay to answer "N/A" if you don't know yet. -->
+
+1. **What related work has already been done in this area?** For decades, OSS have already being sustained by philanthropic funding mostly, but this non-profit approach rarely could effectively sustain consistent efforts in quality assurance, as there aren’t any economical incentives for investors.
+Just recently, **Gitcoin Grants** program were launched with the sole purpose of supporting Public Goods and OSS projects. But as this solution is married with the Quadratic Voting paradigm,  it not only failed to sustain independent OSS projects but also bringed more security issues as this QF model gives perverse incentives for attackers that could steal funds by manipulating the voting system with fake projects.
```

Comment:
```suggestion
Just recently, **Gitcoin Grants** program were launched with the sole purpose of supporting Public Goods and OSS projects. But as this solution is married with the Quadratic Voting paradigm,  it not only failed to sustain independent OSS projects but also brought more security issues as this QF model gives perverse incentives for attackers that could steal funds by manipulating the voting system with fake projects.
```

Reply:
Thanks Ashwin. You're very kind in guiding us through this process. We will fix those typos as soon as possible in the next two days.